pyproject.toml

```diff
@@ -8,12 +8,11 @@ description = "Network Administration Visualized - A comprehensive, free Network
 authors = [{name="Sikt - Norwegian agency for shared services in education and research", email="nav-support@uninett.no"}]
 readme = "README.rst"
 requires-python = ">=3.9"
-license = {text = "GPLv3"}
+license = "GPL-3.0-only"
```

Comment:
This is not as per PEP 639, that would be:

`license-expression = "GPL-3.0-only"`

and removing `license`

however this is not compatible with setuptools.

We cannot merge this until setuptools is 639-compliant, or we could switch to hatchling + versioningit.

Reply:
Re: https://github.com/pypa/setuptools/issues/4629

Reply:
setuptools pushes metadata that pypi does not accept. Wrong version for the fields sent. Anything installing that uses the same validator pypi uses will also croak. That may be fixable by upgrading "packaging" if uploading via "twine".

There's a reason I made a PR for using hatchling instead of setuptools in Argus: https://github.com/pypa/setuptools/issues/4629